automation/src/test/scala/org/broadinstitute/dsde/test/api/WorkspaceApiSpec.scala

```diff
@@ -193,6 +196,106 @@ class WorkspaceApiSpec extends TestKit(ActorSystem("MySpec")) with FreeSpecLike
+            val attributeUpdates = attributeMap.map(attrTuple => AddUpdateAttribute(attrTuple._1, attrTuple._2)).toList
+
+            Rawls.workspaces.updateAttributes(projectName, workspaceName, attributeUpdates)
+            eventually {
+              workspaceResponse(Rawls.workspaces.getWorkspaceDetails(projectName, workspaceName)).workspace.attributes should be (attributeMap)
+            }
+          }
+        }
+      }
+
+      "to delete workspace attributes" - {
+        "from the top" in {
+          implicit val token: AuthToken = ownerAuthToken
+          withCleanBillingProject(owner) { projectName =>
+            withWorkspace(projectName, prependUUID("delete-from-the-top"), attributes = Some(testAttributes)) { workspaceName =>
+              val attributeNameToRemove = getAttributeName("A")
+              val deleteTopAttribute = List(RemoveAttribute(attributeNameToRemove))
+
+              Rawls.workspaces.updateAttributes(projectName, workspaceName, deleteTopAttribute)
+              eventually {
+                workspaceResponse(Rawls.workspaces.getWorkspaceDetails(projectName, workspaceName)).workspace.attributes should be (attributeMap.filter(attribute => attribute._1.name != "A-key"))
+              }
+            }
+          }
+        }
+
+        "from the middle" in {
+          implicit val token: AuthToken = ownerAuthToken
+          withCleanBillingProject(owner) { projectName =>
+            withWorkspace(projectName, prependUUID("delete-from-the-middle"), attributes = Some(testAttributes)) { workspaceName =>
+              val attributeNameToRemove = getAttributeName("B")
+              val deleteMiddleAttribute = List(RemoveAttribute(attributeNameToRemove))
+
+              Rawls.workspaces.updateAttributes(projectName, workspaceName, deleteMiddleAttribute)
+              eventually {
+                workspaceResponse(Rawls.workspaces.getWorkspaceDetails(projectName, workspaceName)).workspace.attributes should be (attributeMap.filter(attribute => attribute._1.name != "B-key"))
+              }
+            }
+          }
+        }
+
+        "from the bottom" in {
+          implicit val token: AuthToken = ownerAuthToken
+          withCleanBillingProject(owner) { projectName =>
+            withWorkspace(projectName, prependUUID("delete-from-the-bottom"), attributes = Some(testAttributes)) { workspaceName =>
+              val attributeNameToRemove = getAttributeName("C")
+              val deleteBottomAttribute = List(RemoveAttribute(attributeNameToRemove))
+
+              Rawls.workspaces.updateAttributes(projectName, workspaceName, deleteBottomAttribute)
+              eventually {
+                workspaceResponse(Rawls.workspaces.getWorkspaceDetails(projectName, workspaceName)).workspace.attributes should be (attributeMap.filter(attribute => attribute._1.name != "C-key"))
+              }
+            }
+          }
+        }
+
+        "after adding them" in {
```

Comment:
same with this one - it's testing the ui table to make sure the ui facet loads correctly.